tests/containers/award/AwardContainer-test.jsx

```diff
@@ -0,0 +1,475 @@
+                type: "C",
+                type_description: "Delivery Order"
+            }
+        }
+    ],
+    total_metadata: {
+        count: 1
+    }
+};
+
+describe('AwardContainer', () => {
+    it('should make an API call for the selected award', () => {
+        // Set up container with mocked award action
+        const awardContainer = shallow(
+            <AwardContainer
+                params={parameters}
+                award={award} />);
+
+        const getAwardSpy = sinon.spy(awardContainer.instance(),
+            'getSelectedAward');
+
+        // Retrieve award and transactions
+        awardContainer.instance().getSelectedAward();
+
+        // checking that it ran
+        expect(getAwardSpy.callCount).toEqual(1);
+
+        // reset the spies
+        getAwardSpy.reset();
+    });
+
+    it('should make an API call for the selected award transactions', () => {
+        // Set up container with mocked award action
+        const awardContainer = shallow(
+            <AwardContainer
+                params={parameters}
+                award={award} />);
+
+        const getTxnSpy = sinon.spy(awardContainer.instance(),
+            'fetchTransactions');
+
+        // Retrieve award and transactions
+        awardContainer.instance().fetchTransactions();
+
+        // checking that it ran
+        expect(getTxnSpy.callCount).toEqual(1);
+
+        // reset the spies
+        getTxnSpy.reset();
+    });
+
+    // parse award
+    it('should parse the returned award and send to redux store', () => {
+        // Set up container with mocked award action
+        const awardContainer = shallow(
+            <AwardContainer
+                params={parameters}
+                award={award}
+                setSelectedAward={AwardActions.setSelectedAward} />);
+
+        const parseAwardSpy = sinon.spy(awardContainer.instance(),
+            'parseAward');
+
+        // Parse award
+        awardContainer.instance().parseAward(fullData);
+
+        // checking that it ran
+        expect(parseAwardSpy.callCount).toEqual(1);
+
+        // reset the spies
+        parseAwardSpy.reset();
+    });
+
+    // parse transactions
+    it('should parse the returned transactions and send to redux store', () => {
+        // Set up container with mocked award action
+        const awardContainer = shallow(
+            <AwardContainer
+                params={parameters}
+                award={award}
+                setAwardTransactions={AwardActions.setAwardTransactions}
+                appendAwardTransactions={AwardActions.appendAwardTransactions}
+                setTransactionsMeta={AwardActions.setTransactionsMeta}
+                updateTransactionRenderHash={AwardActions.updateTransactionRenderHash} />);
+
+        const parseTxnSpy = sinon.spy(awardContainer.instance(),
+            'parseTransactions');
+
+        // Parse award
+        awardContainer.instance().parseTransactions(txnData);
```

Comment:
Same as above; you need to actually test that `parseTransactions` is correctly parsing.